docroot/themes/humsci/humsci_basic/theme-settings.php

```diff
@@ -121,6 +121,22 @@ function humsci_basic_form_system_theme_settings_alter(array &$form, FormStateIn
     '#description' => t("Last line full width option."),
   ];
 
+  // Local Footer
+  $form['options_settings']['humsci_basic_local_footer'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Local Footer Settings'),
+  ];
+
+  $form['options_settings']['humsci_basic_local_footer']['local_footer_variant_classname'] = [
+    '#type' => 'select',
+    '#title' => t('Local Footer Variant'),
+    '#options' => [
+      'default' => '- Default -',
+      'dark' => t('Dark'),
+    ],
+    '#default_value' => theme_get_setting('local_footer_variant_classname'),
+  ];
+
```

Comment:
Question here:

Currently this theme setting is in `humsci_basic` so it's providing the theme setting to all the sub-themes (including traditional and airy). That's been fine for the decanter components (brand bar / global footer), but I'm guessing it's not what we want for this (though we should probably check with Alex).

If this is supposed to be colorful specific, we need to move some of these things into the colorful sub theme.

Reply:
I think this makes sense. I moved the variant to the colorful theme settings and tested the change. I installed the Airy theme and confirmed that:

- The variant only shows up for the Colorful theme
- Other themes do not break when switching themes. The airy theme gets the default class only.